hardware_interface/include/eps.h

```diff
@@ -107,5 +105,6 @@ void prv_instantaneous_telemetry_letoh (eps_instantaneous_telemetry_t *telembuf)
 // If changing the two functions below, update system tasks, too.
 int8_t eps_get_pwr_chnl(uint8_t pwr_chnl_port);
 void eps_set_pwr_chnl(uint8_t pwr_chnl_port, bool bit);
+double csp_letohd(double d);

```

Comment:
This seems like the wrong spot for this. Can we put this function declaration in our fork of libcsp/csp_endian.h?

Reply:
No, for some reason it gets messed up either when passing the argument or passing the return value!